kiwi/bootloader/config/grub2.py

```diff
@@ -271,12 +270,14 @@ def setup_disk_boot_images(self, boot_uuid, lookup_path=None):
             by the distribution. The bios core image is created
             when grub2-install is called
         """
-        log.info('Creating grub bootloader images')
+        self.__copy_theme_data_to_boot_directory(lookup_path)
 
-        if self.firmware.efi_mode():
-            self.efi_boot_path = self.create_efi_path()
+        if not self.firmware.efi_mode():
+            return
 
-        self.__copy_theme_data_to_boot_directory(lookup_path)
+        log.info('Creating grub bootloader images')
+
+        self.efi_boot_path = self.create_efi_path()
 
```

Comment:
sorry I don't get it, the current code does

1. if efi mode; create the standard path
2. copy the theme data
3. if efi mode is standard efi; build the efi image (yes that needs adaptions for other archs but not ppc now)
4. if efi mode is secure boot; get the prebuilt secure boot images

For me there is no reason to add another condition which just returns from the function and again why are we messing with the efi setup when we want to support ppc where we set the firmware to 'ofw'

sorry ??